cmd/syncthing/main.go

```diff
@@ -376,13 +372,12 @@ func syncthingMain() {
 		runtime.GOMAXPROCS(runtime.NumCPU())
 	}
 
-	events.Default.Log(events.Starting, map[string]string{"home": confDir})
+	events.Default.Log(events.Starting, locations)
```

Comment:
Actually no, I'll keep this as it is (`map[string]string{"home": baseDir["config"]}`) instead, there's no reason to change it; it was just a reflex as I was cleaning up some other uses of `confDir`.